### PR TITLE
Update replica field

### PR DESCRIPTION
**Pull request by mustafaStakater**
Update replica field

### CODE REVIEW

**Comment by stakater-user** [comment]
@mustafaStakater Validation Successful

**Review by hanzala1234 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -546,6 +546,8 @@ To disable liveness or readiness probe, set value of `enabled:` to `false`.
 
 All notable changes to this project will be documented here
 
+## v1.1.12
+- Feature: replica field is made optional
```
Comment (rasheedamir) [inline README.md:550]: @mustafaStakater plz add an empty line

**Review by hanzala1234 — DISMISSED** [review]

**Comment by stakater-user** [comment]
@hanzala1234 Validation Successful

File: application/values.yaml [hunk 2]
```diff
@@ -93,7 +93,7 @@ deployment:
     #   timeFormat: world
 
   # Replicas to be created
-  replicas: 2
+  replicas:
```
[the remark this answers is not in the document]
Comment (mustafaStakater) [reply application/values.yaml:96]: I've tested it locally and if there's no value in the replica field, helm doesn't include it in deployment yaml.

[the remark this answers is not in the document]
Comment (rasheedamir) [reply application/values.yaml:96]: so, there is no replica in that case? is there some default value that comes in effect automatically?

[the remark this answers is not in the document]
Comment (hanzala1234) [reply application/values.yaml:96]: we are following the convention to remove only values but keep the key as it is

[the remark this answers is not in the document]
Comment (hanzala1234) [reply application/values.yaml:96]: where there is no replica is spec, replica 1 gets automatically added

[the remark this answers is not in the document]
Comment (mustafaStakater) [reply application/values.yaml:96]: yes, 1 is default replica value in case of no replicas specified

**Comment by rasheedamir** [comment]
@mustafaStakater plz resolve conflicts in README

**Review by AsfaMumtaz — APPROVED** [review]

**Review by hanzala1234 — APPROVED** [review]

**Pull request merged**